templates/pyramid/files/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pyramid = "*"
```

Comment:
Should this also include config-reader, so the user doesn't have to think about that later when they start using a database?